testing-guidance/workflow-tests/o3de-project-manager/remote-templates/remote-templates-workflow-tests.md

```diff
@@ -2,9 +2,18 @@
 
 These workflows center around testing the basic functionality of the Project Manager remote template acquisition tools.
 
+## Documentation & Tutorials
+
+* [Workshop: Building a Networked Game from Scratch...](https://youtu.be/4f4olmUo44k)
+
 ## Prerequisites
 
-*   Locally installed tools for the remote source being used (git installed with credentials for GitHub, for example)
+* Locally installed tools for the remote source being used (git installed with credentials for GitHub, for example)
```

Comment:
Is Git required? I think ProjectManager just reads the URL to see where the zip is saved, and then downloads a zip. It doesn't actually update your pc to store the project in source control.

Reply:
Had a discussion with @AMZN-Gene about this.

It's probably a safe assumption anybody using these workflow docs has git installed and configured. Additionally, depending on repository configurations, a git config may be required if anonymous clones are not allowed and could cause issues with git working with the project manager to perform local user auth for a local clone.